src/core/storage/fileio/general_fstream.cpp

```diff
@@ -1,13 +1,14 @@
 /* Copyright © 2017 Apple Inc. All rights reserved.
  *
  * Use of this source code is governed by a BSD-3-clause license that can
- * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ * be found in the LICENSE.txt file or at
```

Comment:
I'd really prefer not add a new line here. Let's keep all of our copyright headers the same. At some point we might want to right a script to change our headers and I could see something like this breaking that.

Reply:
Or we can run clang-format on all of our files to make this exception to be normal. Hoyt had some scripts adding headers and those scripts work on this kind of scenario. Again, if adding headers, a program should be robust for block and line comments.